javaee/api/src/main/java/org/jboss/forge/addon/javaee/JavaEEFacet.java

```diff
@@ -20,8 +20,8 @@
 {
    String DEFAULT_SERVICE_PACKAGE = "service";
    String DEFAULT_VIEW_PACKAGE = "view";
-   String DEFAULT_CONVERTER_PACKAGE = "converter";
-   String DEFAULT_VALIDATOR_PACKAGE = "validator";
+   String DEFAULT_CONVERTER_PACKAGE = DEFAULT_VIEW_PACKAGE + "." + "converter";
```

Comment:
I think we should rename these constants to DEFAULT_JSF_CONVERTER and DEFAULT_JSF_VALIDATOR, wdyt?